ui/tests/acceptance/volumes-list-test.js

```diff
@@ -69,15 +69,17 @@ module('Acceptance | volumes list', function(hooks) {
 
     const volumeRow = VolumesList.volumes.objectAt(0);
 
-    const controllerHealthStr = volume.controllersHealthy > 0 ? 'Healthy' : 'Unhealthy';
+    const controllerHealthStr = volume.controllersHealthy > 0 ?
+      `Healthy (${volume.controllersHealthy}/${volume.controllersExpected})` :
+      volume.controllerRequired ?
+        'Unhealthy (${volume.controllersHealthy}/${volume.controllersExpected})' :
```

Comment:
Since this string is using single-quotes, the template syntax is being interpreted as literal values. Only back-tick strings are templated.

This either means this test isn't exercising this code path or the test is flaky (only exercising it when the mirage pseudo-randomness creates models that fall into this condition).